testlib/build.gradle.kts

```diff
@@ -28,34 +28,39 @@ import com.google.protobuf.gradle.generateProtoTasks
 import com.google.protobuf.gradle.protobuf
 import io.spine.internal.dependency.Guava
 import io.spine.internal.dependency.JUnit
+import io.spine.internal.dependency.Kotest
 import io.spine.internal.dependency.Protobuf
 import io.spine.internal.dependency.Truth
 import io.spine.internal.gradle.protobuf.setup
 
 group = "io.spine.tools"
 
 plugins {
-    id("detekt-code-analysis")
+    `detekt-code-analysis`
 }
 
 dependencies {
     /*
         Expose tools we use as transitive dependencies to simplify dependency
         management in subprojects.
     */
-    Protobuf.libs.forEach { api(it) }
-    JUnit.api.forEach { api(it) }
-    Truth.libs.forEach { api(it) }
-    api(Guava.testLib)
-    api(kotlin("test-junit5"))
+    (Protobuf.libs
+            + JUnit.api
+            + Truth.libs
+            + Guava.testLib
+            + kotlin("test-junit5")
+            + Kotest.assertions).forEach {
+        api(it)
+    }
```

Comment:
As an alternative, we could have an extension upon `dependencies` block to say: `api(Protobuf.libs)`.

Reply:
The thing here is that we deal with the ”soup” of `String` and `List<String>` here, which is nicely handled by `plus()` operator. 

We can probably extend `api` to handle things, but I would leave it to other PR _and_ when we have more cases like this.